schema/resource_test.go

```diff
@@ -438,7 +438,7 @@ var calculateCQIDPrimaryKeyTestCases = []struct {
 			"MacAddressValue":      "aa:bb:cc:dd:ee:ff",
 			"MacAddressArrayValue": []string{"aa:bb:cc:dd:ee:ff", "11:22:33:44:55:66"},
 		},
-		ExpectedValue:     &UUID{Bytes: [16]uint8{0xa6, 0x76, 0x76, 0xfb, 0x4c, 0x95, 0x51, 0x2d, 0x9e, 0xcd, 0xf4, 0xc4, 0xae, 0xc1, 0x2a, 0xf5}, Status: 0x2},
+		ExpectedValue:     &UUID{Bytes: [16]uint8{0x82, 0xaa, 0xa2, 0xc7, 0x75, 0x10, 0x5c, 0x6d, 0xb8, 0xef, 0x18, 0x49, 0x33, 0x99, 0x4a, 0x9d}, Status: 0x2},
```

Comment:
If this PR is backwards compatible I would expect existing tests to pass. Is this related to the change in `testdata/testdata.go`?

Reply:
I think it is backward compat the the binary can read older encoding but the encoding for the binary types moving forward will be base64. I don't think it should impact anything though.